docs/podtemplates.md

```diff
@@ -95,6 +95,40 @@ Pod templates support fields listed in the table below.
 	</tbody>
 </table>
 
+## Use <code>imagePullSecrets</code> to lookup entrypoint
+
+If no command is configured in <code>task</code> and <code>imagePullSecrets</code> is configured in <code>podTemplate</code>, the Tekton Controller will look up the entrypoint of image with <code>imagePullSecrets</code>. If the Tekton controller's service account is not granted the access to secrets in different namespace. You need to grant the access via RoleBinding:
```

Comment:
Is this necessary, since the SA already has cluster-wide read access to secrets?

If this is meant to document what's necessary for folks that explicitly disable Tekton's cluster-wide secrets access, I think we can just assume that users sophisticated/paranoid enough to do that are able to figure out what else they'd need to do to re-enable it for required secrets.

Reply:
This is in case the access may be removed in the future, Billy told me that it might be removed so it would be better to document it.